src/components/Label/Label.tsx

```diff
@@ -80,27 +84,33 @@ export const Label = React.forwardRef<HTMLDivElement, LabelProps>(function Label
 
     const hasContent = Boolean(children !== '' && React.Children.count(children) > 0);
 
-    const typeDefault = type === 'default';
     const typeClose = type === 'close' && hasContent;
     const typeCopy = type === 'copy' && hasContent;
 
-    const hasOnClick = Boolean(onClick) && typeDefault;
+    const hasOnClick = Boolean(onClick);
     const hasCopy = Boolean(typeCopy && copyText);
     const isInteractive = hasOnClick || hasCopy || interactive;
-    const {copyIconSize, closeIconSize} = sizeMap[size];
+    const {copyIconSize, closeIconSize, buttonSize} = sizeMap[size];
 
     const leftIcon = icon && (
         <div className={b('icon', {side: hasContent ? 'left' : undefined})}>{icon}</div>
     );
     const content = hasContent && <div className={b('text')}>{children}</div>;
 
+    const handleCloseClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+        /* preventing event from bubbling */
+        event.stopPropagation();
```

Comment:
This could affect closing behaviour of Popups. Let's use another way

Reply:
Add hasOnClick check to save old behaviour